app/instructor.py

```diff
@@ -904,6 +920,11 @@ class NewShortAnswerQuestionForm(FlaskForm):
     answer = TextAreaField("Question Answer", [DataRequired()])
     submit = SubmitField("Continue...")
 
+class NewFillInTheBlankForm(FlaskForm):
+    prompt = TextAreaField("Question Prompt", [DataRequired()])
+    answer = TextAreaField("Question Answer", [DataRequired()])
```

Comment:
Remember that there may be multiple blanks to fill in so there will likely need to be multiple text fields for entering the answer.